grakn-test/src/test/java/ai/grakn/test/engine/loader/LoaderTest.java

```diff
@@ -76,19 +79,29 @@ public void setup() {
 
     @Test
     public void loaderDefaultBatchSizeTest() {
-        loadAndTime();
+        loadAndTime(60000);
     }
 
     @Test
     public void loaderNewBatchSizeTest() {
         loader.setBatchSize(20);
-        loadAndTime();
+        loadAndTime(60000);
     }
 
     @Test
     public void loadWithSmallQueueSizeToBlockTest(){
         loader.setQueueSize(1);
-        loadAndTime();
+        loadAndTime(60000);
+    }
+
+    @Test(expected=RuntimeException.class)
```

Comment:
In the future we should probably use the `ExpectedException` class. It accomplishes the same thing, but is a bit neater.